teams/wg-edition-2021.toml

```diff
@@ -0,0 +1,22 @@
+name = "wg-edition-2021"
+subteam-of = "lang"
+kind = "working-group"
```

Comment:
Should this be a working-group or project-group or (sub)team?

Reply:
I believe according to [rfc 2856](https://rust-lang.github.io/rfcs/2856-project-groups.html), working groups are for domains and project groups are for specific efforts. Given that, this should be a project group.

Reply:
Updated!